.github/workflows/notify-on-push.yml

```
@@ -0,0 +1,70 @@
+        run: |
+          gh pr comment ${{ env.PR_ID }} --repo "${{ github.repository }}" \
+            --body "A commit referencing this Pull Request was pushed to `${{ github.ref_name }}` by @${{ github.actor }} with an invalid commit message."
+        env:
+          GH_TOKEN: ${{ github.token }}
+      - name: Slack Notification
+        if: ${{ env.INVALID_COMMIT_MESSAGE }}
+        uses: rtCamp/action-slack-notify@b24d75fe0e728a4bf9fc42ee217caa686d141ee8  # 2.2.1
+        env:
+          SLACK_COLOR: '#DE512A'
+          SLACK_ICON: https://github.com/nodejs.png?size=48
+          SLACK_TITLE: Invalid commit was pushed to ${{ github.repository.default_branch }}
```

actionlint (1.7.4)

receiver of object dereference "default_branch" must be type of object but got "string"

(expression)